StortSpelProjekt/Game/src/Gamefiles/Components/ProjectileComponent.cpp

```diff
@@ -44,6 +47,11 @@ void component::ProjectileComponent::hit(Collision* event)
 		if (event->ent2->HasComponent<component::HealthComponent>())
 		{
 			event->ent2->GetComponent<component::HealthComponent>()->ChangeHealth(-m_Damage);
+			if (event->ent2->GetName().find("conan") != std::string::npos && event->ent2->GetComponent<component::Audio3DEmitterComponent>())
```

Comment:
I am not happy with this hard coded enemy name. If you disagree, dismiss this review